support/helmcharts/jaseci/dev_up

```diff
@@ -0,0 +1,5 @@
+pip install jaseci==1.3.3.17;
+pip install jaseci-serv==1.3.3.17;
```

Comment:
Should the version number here be parameterized for generality?

Reply:
the versioning will be in the prod one, I changed the same. the script file is now a separate file where you can make changes without touching YAMLS